### ISSUE TITLE
Sample variant browser view button for primary findings filtering not working

### ISSUE THREAD

**Issue by pamag** [opencb/jsorolla#340]
## Describe the bug

when using the filter in the view button in the sample variant browser:

![image](https://user-images.githubusercontent.com/167875/149376072-3d3b63af-1c53-49e4-a5be-67751b7e39de.png)

The filter for the variants is added to the end of the current filter but should replace the filter (exept the sample)

![image](https://user-images.githubusercontent.com/167875/149787948-ce4e19b6-def6-4da0-b426-df9b814e882d.png)

If we remove the panel it works fine:

![image](https://user-images.githubusercontent.com/167875/149376344-34b08f48-4c6d-498e-98ec-b7231d0d2400.png)

The panel can be present but then we need to set the `panel intersection ON` before applying the view filter:

![image](https://user-images.githubusercontent.com/167875/149376890-6bdabc0a-8248-4408-8dc8-971aa2378659.png)

## How to reproduce
![view_error_and_panel_intersect](https://user-images.githubusercontent.com/167875/149375803-1d4f9497-74c4-40b9-a8bd-597f5da2f811.gif)

## Proposed Fix

When doing the selected variants view filter, the only filter present in the filter navigation should be the regions of the variants (plus the sample).

![image](https://user-images.githubusercontent.com/167875/149376344-34b08f48-4c6d-498e-98ec-b7231d0d2400.png)

BUT if there is a panel lock (it needs to be kept) be sure to use the panel_intersect=ON and add the variants after the panel

![image](https://user-images.githubusercontent.com/167875/149376890-6bdabc0a-8248-4408-8dc8-971aa2378659.png)

NOTE: if you where in middle of a new filter query it will be lost. Would be nice to show a message in case the current filter set is not the default:
```
Warning: you have a modified filter query. Searching for selected variants will update the filter and your query will be lost. Please save your query in the filter menu if you want to keep it. 
```

**Comment by Rodielm** [comment]
The reason why the `depth` is not removed when running the `initial query` is because of this config in the `opencga-active-filter`.  Especially `sampleData`

The real problem is that if we remove it, this could affect other components that use `opencga-active-filter`.
![image](https://user-images.githubusercontent.com/10471758/151551403-9da1fb03-5355-4d9a-9c6b-b644bdac68f8.png)

This issue will be discussed later on how to solve it without affecting other components.

**Comment by Rodielm** [comment]
Fixed : bb540b6
![variantFilterView2](https://user-images.githubusercontent.com/10471758/151827874-5b11564f-efe5-4232-9238-9d8986ac15b6.gif)

**Comment by pamag** [comment]
## Reviewed

commit: a211bb1e039c4f68309f0436a8d8ce8182da90e4

FIXED
